sync/index.js

```diff
@@ -186,8 +186,6 @@ function* checkSyncStatus() {
   var lastSyncTime;
   if (config.syncModel === 'all') {
     lastSyncTime = total.last_sync_time;
-  } else if (config.syncModel === 'exist') {
-    lastSyncTime = total.last_exist_sync_time;
```

Comment:
他的情况是已有的模块只有10个这种很少的情况，所以同步时间会设置成一天吧。要不就将这个检查时间做成可配置的？或者说要比同步时间间隔大。

Sent from my iPhone

> On Apr 5, 2016, at 11:46 AM, Yiyu He <notifications@github.com> wrote:
> 
> In sync/index.js:
> 
> > @@ -186,8 +186,6 @@ function* checkSyncStatus() {
> >    var lastSyncTime;
> >    if (config.syncModel === 'all') {
> >      lastSyncTime = total.last_sync_time;
> > -  } else if (config.syncModel === 'exist') {
> > -    lastSyncTime = total.last_exist_sync_time;
> exist 的这个模式也会定时去同步现在仓库内已经有了的模块啊
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>